pkg/datasource.go

```diff
@@ -65,16 +68,18 @@ type GrafanaSearchRequest struct {
 	GrafanaCommonRequest
 	Metric        string `json:"metric,omitempty"`
 	Namespace     string
+	TenancyConfig string
 	ResourceGroup string
 }
 
 // GrafanaCommonRequest - captures the common parts of the search and metricsRequests
 type GrafanaCommonRequest struct {
-	Compartment string
-	Environment string
-	QueryType   string
-	Region      string
-	TenancyOCID string `json:"tenancyOCID"`
+	Compartment   string
+	Environment   string
+	QueryType     string
+	Region        string
+	TenancyConfig string
+	TenancyOCID   string `json:"tenancyOCID"`
```

Comment:
For the first point, I do not agree. TenancyID is very similar to TenancyOCID, causing in my opinion potential confusions. Moreovethe variably is not containing a TenancyID rather than really a TenancyConfig.

Reply:
I can live with the field/variable name in the code as long as there is a comment that explains what is actually in the string. However, in the UI the term `TenancyConfig` should not be used, the label should simply be `Tenancy`.

Reply:
I add some comments to explain how the var is formed